de-lib/src/main/java/org/iplantc/de/apps/integration/client/presenter/ToolSearchRPCProxy.java

```diff
@@ -50,11 +51,11 @@ public void load(FilterPagingLoadConfig loadConfig,
         // Cache the search text for this callback; used to sort the results.
         final String searchText = lastQueryText;
         hasHandlers.fireEvent(new BeforeToolSearchEvent());
-        dcService.searchTools(loadConfig, new AsyncCallback<List<Tool>>() {
+        dcService.searchTools(loadConfig, new AppsCallback<List<Tool>>() {
 
             @Override
-            public void onFailure(Throwable caught) {
-                ErrorHandler.post(I18N.ERROR.dcLoadError(), caught);
+            public void onFailure(Integer statusCode, Throwable exception) {
+                ErrorHandler.post(I18N.ERROR.dcLoadError(), exception);
```

Comment:
We could switch this to an appearance class instead of using the I18N.  I've been trying to refactor those calls out.  Small thing though.